CHANGELOG.md

```diff
@@ -3,6 +3,55 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [4.0.0-alpha.20](https://github.com/mui-org/material-ui-x/compare/v4.0.0-alpha.19...v4.0.0-alpha.20)
+
+###### _Feb 16, 2021_
+ 
+Big thanks to the 5 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Add new selectionModel prop (#986) @dtassone
+- 🚀 Fix controllable filters and select all rows with filters (#1020) @dtassone
+- 💅 Add support for default props from theme (#1019) @DanailH
+- 🙌 Fix scrollbar size on windows (#1061) @dtassone
+- ⚡️ Fix server-side multi filters (#1029) @dtassone
+- 🐛 Polish existing features, fix 9 issues.
+
+### @material-ui/x-grid@v4.0.0-alpha.19 / @material-ui/data-grid@v4.0.0-alpha.19
+
+- [DataGrid] Add bgBG locale (#983) @DanailH
```

Comment:
It's really important that we document the breaking changes: #1021, #1015, #986, #948.

```suggestion
#### Breaking changes

#### Changes

- [DataGrid] Add bgBG locale (#983) @DanailH
```

Reply:
Do you want to add the content for these changes?